freqtrade/freqtradebot.py

```diff
@@ -73,7 +73,7 @@ def __init__(self, config: Dict[str, Any]) -> None:
 
         self.active_pair_whitelist: List[str] = self.config['exchange']['pair_whitelist']
 
-        persistence.init(self.config)
+        persistence.init(self.config.get('db_url', None), self.config.get('dry_run', False))
```

Comment:
dry_run is required. so why do you get a default value?

Reply:
Well - tbh - it mimics how it did work before (within `.init()`).

but you're right - i should reconsider...

Reply:
going to merge this as an out of band discussion (slack) with @mishaker  did confirm this as ready to merge.